src/core/reducers/autocomplete.js

```diff
@@ -0,0 +1,69 @@
+export const AUTOCOMPLETE_LOADED = 'core/autocomplete/AUTOCOMPLETE_LOADED';
+export const AUTOCOMPLETE_STARTED = 'core/autocomplete/AUTOCOMPLETE_STARTED';
+export const AUTOCOMPLETE_CANCELLED = 'core/autocomplete/AUTOCOMPLETE_CANCELLED';
+
+const initialState = {
+  loading: false,
+  suggestions: [],
```

Comment:
I haven't looked at the UI patch yet but defaulting `suggestions` to an empty array makes it look no different than an API response of zero results. Maybe that's okay but previously this has caused problems (especially due to server rendering) where we have no way to tell if an initial API request had been made or not. We would have solved this by defaulting `suggestions` to `null` to indicate that no API request has been made. I would suggest trying to work with that model unless you find yourself writing too much defensive code around it.